### ISSUE TITLE
Documentation Links Broken

### ISSUE THREAD

**Issue by shkkmo** [doctrine/common#856]
Many of the links throughout the web to doctrine's documentation have all been broken. This includes a large number of links from blogs and from stackoverflow.

Example:
http://docs.doctrine-project.org/en/latest/reference/advanced-configuration.html

The issue appears to be a re-organization that moved that resource to:

http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/advanced-configuration.html

Instead of showing a "Page not found" error, that resource (and all other moved resources) should provide a 301 redirect response to the new url for that resource.

**Comment by jwage** [comment]
It is unfortunately not possible to do currently because we use a statically built website and github pages. It might be possible to build a map of redirect rules and do it in javascript though.

**Comment by shkkmo** [comment]
Maybe I'm missing something, but GitHub pages should be able to do a
redirect easily:

https://help.github.com/articles/redirects-on-github-pages/

**Comment by Majkl578** [comment]
Would that work with HTML too? We generate HTML from RST manually and only publish final HTMLs on the GH Pages.

**Comment by shkkmo** [comment]
Yes. The article I linked talks about using a plugin for github page's default static build tool, Jekyll, that generates static pages that use the refresh meta tag as an inferior substitute for a 301 redirect.

I don't know the details of your build process, but having the ability to not break links to documentation seems pretty important.

**Comment by Majkl578** [comment]
It's important to note that it also includes different subdomain, which may significantly complicate things.
It may be a lot easier to set up static redirects with NGINX for some period. I'll discuss that with @jwage if there is some sane way to get _old -> new_ URL map to set it up.
